05.wc/wc.rb

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+PRINTING_WIDTH = 8
+
+def main
+  options = ARGV.getopts('lwc')
+  ARGV.none? ? print_pipelined_input(options) : print_specified_files(ARGV, options)
+end
+
+def print_pipelined_input(options)
+  stdin_lines = $stdin.readlines
+
+  stdin_prop = format_file_prop(stdin_lines)
```

Comment:
この場合、line_countやword_countがそれぞれpropで、それをまとめたものをpropsと呼びたいところですが、そうすると「propsが入った配列を何と呼べばいいのか」という問題が出てきますね。

```suggestion
  # 本当はこうしたい
  stdin_props = format_file_props(stdin_lines)
```

こういう場合、何かお決まりの回避策があるわけではないですが、僕は`_list`というsuffixを付けて「〜のリスト（〜の配列）」という意味を持たせたりします。

```ruby
def print_file_props_list(file_props_list, options)
  file_props_list.each do |file_props|
```